### PR TITLE
Add builder for AprilTags

### PR DESCRIPTION
**Pull request by Affie**

### CODE REVIEW

File: A/AprilTags/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    rm -f *.dll
+fi
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:glibc),
+    Windows(:x86_64),
+    MacOS(:x86_64)
```
Comment (giordano) [inline A/AprilTags/build_tarballs.jl:37]:
```suggestion
    Platform("x86_64", "linux"; libc="glibc"),
    Platform("x86_64", "windows"),
    Platform("x86_64", "macos"),
```

File: A/AprilTags/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+export CFLAGS="-std=gnu99"
+
+cmake .. -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DBUILD_PYTHON_WRAPPER=Off
+make -j${nproc}
+make install
+
+# Move dll on windows
+if [[ ${target} == *-mingw32 ]]; then 
+    cp -L *.dll $WORKSPACE/destdir/bin
+    rm -f *.dll
```
Comment (giordano) [inline A/AprilTags/build_tarballs.jl:28]:
```suggestion
    cp -L *.${dlext} ${libdir}
```

File: A/AprilTags/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+fi
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"; libc="glibc"),
+    Platform("x86_64", "windows"),
+    Platform("x86_64", "macos"),
+]
```
Comment (giordano) [inline A/AprilTags/build_tarballs.jl:37]: Any reason for not building for all platforms?
```suggestion
platforms = supported_platforms()
```

Comment (Affie) [reply A/AprilTags/build_tarballs.jl:37]: Hi I tested locally and first wanted to get the basics working. Thanks for the fixes, I used the BinaryBuilder wizard.

**Comment by giordano** [comment]
Looks like they all worked flawless :slightly_smiling_face:  Thanks for the contribution!

**Pull request merged**